proto/backup.proto

```diff
@@ -43,6 +43,11 @@ message BackupMeta {
 
     // In incremental backup, DDLs which are completed in (lastBackupTS, backupTS] will be stored here.
     bytes ddls = 10;
+
+    // Save the br commit to run backup jobs
+    string br_version = 11;
+    // A map of the tikv store ID to store version
+    map<uint64, string> tikv_versions = 12;
```

Comment:
I'm a bit confused with this map. If some of the tikv instances' version are different, what can we do with them?

Reply:
It will be simpler for br to debug. If we fail to ingest an SST file because of the version compatibility problem, we can get  store_id from this file's name and version from this map.

Reply:
Ingest SST happens in restore, but backupmeta is created in backup.

Reply:
When we fail to ingest sst in restore, we can check the `backupmeta` to find out the tikv's version who backuped this sst file.

Reply:
TiKV rarely changes SST file format (still in SST v2). I think it's fine to not record tikv version.

Reply:
tikv_versions deleted in https://github.com/pingcap/kvproto/pull/717/commits/9ee61335654225c1fad27beac7806ef389212d64